feather/server/src/systems/view.rs

```diff
@@ -40,16 +40,20 @@ fn send_new_chunks(game: &mut Game, server: &mut Server) -> SysResult {
         .query::<(&ClientId, &ViewUpdateEvent, &Position)>()
         .iter()
     {
-        let client = server.clients.get(client_id).unwrap();
-        client.update_own_chunk(event.new_view.center());
-        update_chunks(
-            game,
-            player,
-            client,
-            event,
-            position,
-            &mut server.waiting_chunks,
-        )?;
+        // As each iteration takes a while, it can happen that a player
```

Comment:
The cause of the problem isn't actually clients being removed while the loop is running, since the client list is only accessible by one thread. It happens because players in the ECS aren't removed for one extra tick after they disconnect (so plugins can handle disconnect events), and as a result, the query finds dead clients.

The solution here still works, but the comment should be changed.